scripts/common

```diff
@@ -7,13 +7,16 @@ bootstrap(){
 	local BOOTSTRAP_CMD=debootstrap
 	local BOOTSTRAP_ARGS=()
 
-	export http_proxy=${APT_PROXY}
-
+	if proxy_check; then
```

Comment:
Unless I'm missing something, if proxy_check fails, this seems to be disabling it regardless of whether APT_PROXY_FALLBACK is set.

Reply:
Since bootstrap is one of the earliest operations that fetch files, checking proxy in build.sh shadowed the bug in my test environment, I think. Fixed now.